src/spikeinterface/core/sortinganalyzer.py

```diff
@@ -2033,19 +2033,18 @@ def load(cls, sorting_analyzer):
         return None
 
     def load_run_info(self):
+        run_info = None
         if self.format == "binary_folder":
             extension_folder = self._get_binary_extension_folder()
             run_info_file = extension_folder / "run_info.json"
             if run_info_file.is_file():
                 with open(str(run_info_file), "r") as f:
                     run_info = json.load(f)
-            else:
-                warnings.warn(f"Found no run_info file for {self.extension_name}, extension should be re-computed.")
```

Comment:
The warning was here to be aware that the analyzer was computed before this run info.

Reply:
Take a look at the full current implementation here: https://github.com/SpikeInterface/spikeinterface/blob/65d4b1ed28e2972e03edc3796942d7ba4445ab10/src/spikeinterface/core/sortinganalyzer.py#L2035 
If the first warning is triggered, the second is always triggered (I think!). So the first warning isn’t needed, and this is deleted here.

Reply:
I think the second if block was meant to be indented under the `zarr` case, but I agree it works just as well to assign it to None in the first place and then do the warning at the end if it's still None.

Reply:
Yes, I did wonder if it was a missing indentation. This solution's a tiny bit neater as we only need one copy of the warning message. I like the idea of the `run_info` in general: great suggestion @jonahpearl  :)